rl_studio/envs/f1/image_f1.py

```diff
@@ -1,7 +1,7 @@
 import cv2
 import numpy as np
 
-from rl_studio.agents.f1.settings import QLearnConfig
+from agents.f1.settings import QLearnConfig
```

Comment:
Include the absolute path to this class.

Reply:
Done